### PR TITLE
Updating Liberty API/SPI targets for 16.0.0.2 release

### PR DESCRIPTION
**Pull request by rajivsen**
Updated the pom.xml files in the target folders to incorporate the liberty API/SPI changes for the 16.0.0.2 release.

### CODE REVIEW

File: targets/liberty-apis/pom.xml [hunk 1]
```diff
@@ -19,228 +19,234 @@
...
       <artifactId>com.ibm.websphere.appserver.api.distributedMap</artifactId>
-      <version>2.0.12</version>
+      <version>2.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.dynamicRouting</artifactId>
-      <version>1.0.12</version>
+      <version>1.1.13</version>
```
Comment (cbridgha) [inline targets/liberty-apis/pom.xml]: Just checking that the major version also changed?

File: targets/liberty-apis/pom.xml [hunk 2]
```diff
@@ -19,228 +19,234 @@
...
       <artifactId>com.ibm.websphere.appserver.api.restConnector</artifactId>
-      <version>1.1.12</version>
+      <version>1.1.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.saml20</artifactId>
-      <version>1.0.12</version>
+      <version>1.1.13</version>
```
Comment (cbridgha) [inline targets/liberty-apis/pom.xml]: Another major version change?

File: targets/liberty-apis/pom.xml (hunk 1 above)
Comment (rajivsen) [inline targets/liberty-apis/pom.xml]: Yes that's correct:
http://search.maven.org/#search|gav|1|g%3A%22com.ibm.websphere.appserver.api%22%20AND%20a%3A%22com.ibm.websphere.appserver.api.dynamicRouting%22

File: targets/liberty-apis/pom.xml (hunk 2 above)
Comment (rajivsen) [inline targets/liberty-apis/pom.xml]: Yes that's correct:
http://search.maven.org/#search|gav|1|g%3A%22com.ibm.websphere.appserver.api%22%20AND%20a%3A%22com.ibm.websphere.appserver.api.saml20%22

**Pull request merged**